package.json

```diff
@@ -28,6 +28,11 @@
     "mssql": "4.0.4"
   },
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
can `"babel-core": "^6.26.0",` and `"babel-preset-es2015": "^6.24.1",` be moved out of the `dependencies` section as well?

Reply:
Let's ensure to remove the carrots '^' and lock down the versions to the ones we are using in our package-lock.json file.